src/Core/src/ImageSources/IImageSourceServiceResult.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Microsoft.Maui
+{
+	public interface IImageSourceServiceResult<T> : IDisposable
```

Comment:
This type has 2 purposes:
1. a way to "dispose" the image - either by calling Value.Dispose, or by returning it to the bitmap pool/cache
2. a way to indicate that the bitmap needs to be regenerated if the rendering surface changes DPI - such as when the app is moved monitors or the user changes the screen DPI